Replace ADMIN_USERNAME with ADMIN_USER in systemd and containerized i… (#183)
SUSE/doc-unversioned: trento/migration/container-install.md

1. Install trento-web on Docker

>    Be sure to change the `ADMIN_USER` and `ADMIN_PASSWORD`, these are the credentials that will be required to login to the trento-web UI.

Reviewer: ```suggestion
   Make sure to change the `ADMIN_USER` and `ADMIN_PASSWORD`. These credentials are required to login to the trento-web UI.
```
